App.js

```diff
@@ -1,12 +1,14 @@
 import { StatusBar } from 'expo-status-bar';
-import React from 'react';
+import React, { Component } from 'react';
 import { StyleSheet, Text, View } from 'react-native';
+import Cat from './Cat';
 
 export default function App() {
   return (
     <View style={styles.container}>
       <Text>Hello, world!</Text>
       <StatusBar style="auto" />
+      <Cat></Cat>
```

Comment:
I noticed my code I just have <Cat /> instead of <Cat></Cat>. Not sure what the difference is, but they both seem to work. Great stuff!